requirements/common.txt

```diff
@@ -313,3 +253,16 @@ iso8601==0.1.12 \
 
 enum34==1.1.6; python_version < '3' \
     --hash=sha256:6bd0f6ad48ec2aa117d3d141940d484deccda84d4fcd884f5c3d93c23ecd8c79
+
+python-jose==2.0.2 \
+    --hash=sha256:3b35cdb0e55a88581ff6d3f12de753aa459e940b50fe7ca5aa25149bc94cb37b
+
+# Used by python-jose
+pycryptodome==3.4.8 \
+    --hash=sha256:e089d8619a46212cc191258292a3ffbe0d44b9e5ca21f036a4ca8f2db6b0860a
```

Comment:
This package is distributed as a Python wheel containing compiled code (to reduce install time), so has multiple variants of the package available for download for each OS/Python version combination. As such, it also needs the Python 3 wheel's hash listed to make the Python 3 Travis job pass:
`2c1ca8e134dc812555400e0a993cd4b9c858c968e6819522ae69d7b0ad6437a7`

(We're not using Python 3 yet, but have a sanity-check travis job to gradually increase coverage on the progress made in the transition)

Reply:
I added this extra hash but it's still complaining about a missing hash.

Reply:
I think that's because the line continuation marker (`\`) is missing?

Reply:
Ugh, my bad.